ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+# Description of problem
+
+(remove this text and list the steps you followed)
+
+# Expected result
+
+(remove this text and explain what you thought would happen)
+
+# Actual result 
+
+(remove this text and explain what actually happened)
+
+---
+
+(replace this text with the output of the `cc-collect-data.sh` script, after you have
```

Comment:
we could hotlink to the `cc-collect-data.sh` script as well

Reply:
Good idea. Updating...

Reply:
Ah - slight complication: https://github.com/clearcontainers/runtime/pull/606 changes the script to be auto-generated. So we can point to the unbuilt script, but not the script itself. What do you think?

Reply:
heh. Oh. Maybe we can note something like 'that came with your installation, or is available after a build from source' or similar? But, fine if we leave as-is.